src/ReactWrapper.js

```diff
@@ -471,15 +471,15 @@ export default class ReactWrapper {
    * @param {Array} args
    * @returns {ReactWrapper}
    */
-  simulate(event, ...args) {
+  simulate(event, mock) {
```

Comment:
since the mock is optional, it should probably explicitly default to a value - that way `simulate`'s `length` won't change.